src/jdk.jdwp.agent/share/native/libjdwp/util.c

```diff
@@ -1901,9 +1908,44 @@ map2jvmtiError(jdwpError error)
     return AGENT_ERROR_INTERNAL;
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static jvmtiExtensionEventInfo *
+find_ext_event(jvmtiEnv *jvmti, const char *ename)
+{
+    jint extCount = 0;
+    jvmtiExtensionEventInfo *extList = NULL;
+
+    jvmtiError err = JVMTI_FUNC_PTR(jvmti, GetExtensionEvents)
+                            (jvmti, &extCount, &extList);
+    if (JVMTI_ERROR_NONE == err) {
+        for (int i = 0; i < extCount; i++) {
+            if (NULL != strstr(extList[i].id, ename)) {
```

Comment:
A better name for `ename` is `eventName`.
This should use `strcmp()`.
In C code, locals cannot be declared in `for` loops.
```c
find_ext_event(jvmtiEnv *jvmti, const char *eventName)
{
    jint extCount = 0;
    jvmtiExtensionEventInfo *extList = NULL;

    jvmtiError err = JVMTI_FUNC_PTR(jvmti, GetExtensionEvents)
                            (jvmti, &extCount, &extList);
    if (JVMTI_ERROR_NONE == err) {
        jint i = 0;
        for (i = 0; i < extCount; i++) {
            if (0 == strcmp(extList[i].id, eventName)) {
```

Reply:
Updated, and also avoid multiple returns.